src/frame/mod.rs

```diff
@@ -36,6 +109,11 @@ pub(crate) type Word = u16;
 /// Number of items to process.
 pub type Quantity = u16;
 
+/// A Trait that is implemented on types that have a function code
```

Comment:
This trait is not needed. Could be implemented independently on types using `const fn` methods.

Reply:
I've deleted the trait